.github/workflows/test.yml

```diff
@@ -96,6 +96,7 @@ jobs:
           cd triton/
           uv pip install -r python/requirements.txt
           MAX_JOBS=$(nproc) TRITON_PARALLEL_LINK_JOBS=2 uv pip install .
+          python -c "import triton; print(f'Triton version: {triton.__version__}')"
```

Comment:
actually I wonder if there is already a “triton” folder within the Triton checkout, which causes `import triton` to always work… maybe to be safer, we can move this to be after `rm -rf triton/` line.